### PR TITLE
Hotfix add matching funds as donations

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
Related to #1186

### CODE REVIEW

**Comment by mohammadranjbarz** [comment]
I just added a button in admin panel to sync donations with distributed matching funds, but we should add some cronjobs for that to call the function daily, or something like that

**Relate Donations with Distributed Funds** button
Screen Shot 1402-09-14 at 16 02 46

File: migration/1701689359018-add_fields_to_qf_round_history.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    // Alpha round in production
+    await queryRunner.query(`
+        UPDATE qf_round_history 
+        SET 
+            "matchingFundAmount" = "matchingFund", 
+            "matchingFundPriceUsd" = 1, 
+            "matchingFundCurrency" = 'WXDAI'
+        WHERE 
+            id = 2 AND "matchingFund" IS NOT NULL;
```
Comment (jainkrati) [inline migration/1701689359018-add_fields_to_qf_round_history.ts:35]: why are we hard coding the id to 2?

File: src/services/donationService.ts [hunk 2]
```diff
@@ -436,3 +444,91 @@ export const sendSegmentEventForDonation = async (params: {
...
+    );
+    return;
+  }
+  logger.debug(
+    `insertDonationsFromQfRoundHistory Filling ${qfRoundHistories.length} qfRoundHistory info ...`,
+  );
+
+  const matchingFundFromAddress =
+    (process.env.MATCHING_FUND_DONATIONS_FROM_ADDRESS as string) ||
+    '0x6e8873085530406995170Da467010565968C7C62'; // Address behind donation.eth ENS address;
```
Comment (jainkrati) [inline src/services/donationService.ts]: lets take this as a constant value

**Review by jainkrati — CHANGES_REQUESTED** [review]
Left a comment. Once resolved we are good to merge

File: migration/1701689359018-add_fields_to_qf_round_history.ts (hunk 1 above)
Comment (mohammadranjbarz) [reply migration/1701689359018-add_fields_to_qf_round_history.ts:35]: We have some qf rounds already in our DB and they have qfRoundHistory items in DB, they dont have new fields, so I had to put their ids hard coded here

Comment (mohammadranjbarz) [reply migration/1701689359018-add_fields_to_qf_round_history.ts:35]: Screen Shot 1402-09-14 at 17 10 15

@jainkrati

**Review by jainkrati — APPROVED** [review]

**Pull request merged**